package.json

```diff
@@ -8,7 +8,16 @@
     "start": "react-native start",
     "test": "jest",
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
-    "clean-up": "rm -rf .git README.md && rm -rf ./assets/RN-Typescript-Boilerplate.gif ./assets/logo.png && git init"
+    "clean-up": "rm -rf .git README.md && rm -rf ./assets/RN-Typescript-Boilerplate.gif ./assets/logo.png && git init",
+    "postinstall": "jetify",
```

Comment:
I think this should be better like: 
`"post-install": "npx jetify"`

Reply:
Hmm npx right will update this.